### ISSUE TITLE
Interactive CLI report to report version number

### ISSUE THREAD

**Issue by wolfgangrumpf** [fireice-uk/xmr-stak#2492]
**Please explain the feature as good as possible.**
I often leave XMR-stak running for weeks at a time.  And I don't always update to every release.  It would be nice if, in the running instance, we had an additional command to show the current version of xmrstak as well as the coin being mined.  We currently have:

r = results
h = hash report 
c = connection report

How about:

v = version info (e.g. xmr-stak 10.1, currently mining monero)

**Comment by qtweeder** [comment]
it shows the version in the web interface, and also at the top of the stak command instance (scroll to the top)

**Comment by wolfgangrumpf** [comment]
Yes, it does show the version at the top of the stak command instance.  I launch my instances in a screen session, which limits scrollback.  I suppose I could launch them headless and send the output to a logfile instead so that I could scroll....but even just adding the version to one of the existing commands would make things "easier".
